src/web/components/Section.tsx

```diff
@@ -3,7 +3,7 @@ import { css } from '@emotion/react';
 import { border } from '@guardian/src-foundations/palette';
 import { from } from '@guardian/src-foundations/mq';
 
-const center = css`
+export const center = css`
```

Comment:
Is there somewhere for using shared CSS, rather than exporting it from another component?

Reply:
I don't think we have anywhere, but one of the `lib` directories is probably a good choice here - I'll move it.